shawmut/weather.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from requests_forecast import Forecast
+from shawmut.settings import conf
+
+class ShawmutWeather(object):
+    MAX_RETRIES = 500 # We are allowed 1000 API calls per today so this is pretty conservative
+
+    def __init__(self):
+        self.refresh_todays_data()
+
+    def get_todays_data(self):
```

Comment:
I think you can combine this with `refresh_todays_data` -- `get_todays_data` implies that it should return a data object, and it does not. You can just wrap all of `refresh_todays_data` in this if statement.